ipengine_config.py

```diff
@@ -407,10 +393,10 @@
 # Configurable for MPI initialization
 
```

Comment:
This is the bit that I added to initialize MPI. On this machine the engines  will abort if they don't initiate  mpi communication.